CHARTER.md

```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
 
    - v. appointing representatives to work with other open source or open standards communities;
 
-   - vi. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
 
-   - vii. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the Technical Initiative; and
+ - f. The working group lead is responsible of
```

Comment:
There's a section for the responsibilities for the lead, but not for the co-lead. Probably should add that.